Content.Server/GameTicking/Rules/Components/NukeopsRuleComponent.cs

```diff
@@ -44,6 +41,54 @@ public sealed class NukeopsRuleComponent : Component
     [DataField("spawnOutpost")]
     public bool SpawnOutpost = true;
 
+    /// <summary>
+    /// Whether or not nukie left their outpost
+    /// </summary>
+    [DataField("leftOutpost")]
+    public bool LeftOutpost = false;
+
+    /// <summary>
+    ///     Enables opportunity to get extra TC for war declaration
+    /// </summary>
+    [DataField("enableWarOps")]
+    public bool EnableWarOps = true;
+
+    /// <summary>
+    ///     Indicates time when war has been declared, null if not declared
+    /// </summary>
+    [DataField("warDeclaredTime")]
+    public TimeSpan? WarDeclaredTime;
+
+    /// <summary>
+    ///     Delay between declaration of war and enabling FTL at outpost map
+    /// </summary>
+    [DataField("warShuttleDelay")]
+    public bool PreventShuttleInDelay = true;
```

Comment:
Can't you just make WarNukieArriveDelay a nullable timespan instead and remove this?